bot/tests/test_reporter_phabricator.py

```diff
@@ -627,17 +631,19 @@ def test_phabricator_unitresult(mock_phabricator, phab, mock_try_task, mock_task
         assert phab.build_messages["PHID-HMBD-deadbeef12456"] == [
```

Comment:
We need to check the summary comment here too. We have no idea how the coverity issues are listed now